UI/Navigation/NavigationWithXaml/NavigationWithXaml/Presentation/MainPage.xaml

```diff
@@ -0,0 +1,32 @@
+﻿<Page x:Class="NavigationWithXaml.Presentation.MainPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:local="using:NavigationWithXaml.Presentation"
+      xmlns:uen="using:Uno.Extensions.Navigation.UI"
+      xmlns:utu="using:Uno.Toolkit.UI"
+      xmlns:um="using:Uno.Material"
+      NavigationCacheMode="Required"
+      Background="{ThemeResource BackgroundBrush}">
+  <ScrollViewer>
+    <Grid utu:SafeArea.Insets="VisibleBounds">
+      <Grid.RowDefinitions>
+        <RowDefinition Height="Auto" />
+        <RowDefinition />
+      </Grid.RowDefinitions>
+      <utu:NavigationBar Content="{Binding Title}" />
+
+      <StackPanel Grid.Row="1"
+            HorizontalAlignment="Center"
+            VerticalAlignment="Center"
+            Spacing="16">
+        <Image Width="150"
+               Height="150"
+               Margin="12"
+               HorizontalAlignment="Center"
+               Source="ms-appx:///Assets/logo.png" />
+        <Button Content="Go to SamplePage"
+                uen:Navigation.Request="Sample" />
```

Comment:
Maybe we could add a simple comment explaining stating that navigation is being triggered by `uen:Navigation.Request` ?